lib/renuo_cms_rails/cms_helper.rb

```diff
@@ -0,0 +1,28 @@
+require 'action_view/helpers'
+
+module RenuoCmsRails
+  module CmsHelper
+    def cms(path, default_value = nil, &block)
+      url_path = "#{path}-#{I18n.locale}"
```

Comment:
Appending a locale string is a convention which maybe should be communicated. Also this makes me ask how paths should be named. Apparently paths will look like this in the end: `view.article.index.intro-text-de-CH`